apps/prairielearn/src/api/v1/index.ts

```diff
@@ -11,12 +10,12 @@ const router = Router();
  * viewing permissions. This should be added to any routes that provide student
  * data.
  */
-const authzHasCourseInstanceView = asyncHandler(async (req, res, next) => {
+const authzHasCourseInstanceView = (req: Request, res: Response, next: NextFunction) => {
   if (!res.locals.authz_data.has_course_instance_permission_view) {
     throw new error.HttpStatusError(403, 'Requires student data view access');
   }
   next();
-});
+};
```

Comment:
This is a change in behavior. asyncHandler takes care of the exception, without it you need to call next with the exception as an argument.